### DISCUSSION TITLE
Full text search apps removed after manual installation

### DISCUSSION THREAD

**Discussion by verement** [nextcloud/all-in-one#3661] [Questions] [answered]
### Steps to reproduce
1. Start Nextcloud AIO containers **without** *Fulltextsearch* enabled
2. Manually download and enable these apps:
    - *Full text search*
    - *Full text search - Elasticsearch Platform*
    - *Full text search - Files*
3. (Configure *Full text search* to use an existing Elasticsearch installation)
4. Wait until the daily automatic backup completes
5. Observe that the three apps above have been removed

### Expected behavior 
The manually installed apps should not be removed.

### Actual behavior 
The manually installed apps listed above are disabled and removed.

### Host OS 
Debian 12.2

#### Nextcloud AIO version 
v7.5.1

#### Current channel 
latest

#### Other valuable info 
Manually installing the full text search apps rather than using the AIO-provided Fulltextsearch should be a valid option because one may already have an Elasticsearch installation one would like to use without expanding the resource requirements of a second instance.

Thank you.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-keep-disabled-apps

**Reply by verement** [reply]
Thank you for the pointer, and apologies for the noise.
